core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/EntityScopeProviderImpl.java

```diff
@@ -240,21 +233,19 @@ public Scope getScopeForInterfaceByIfIndex(final Integer nodeId, final int ifInd
 
             // SNMP interface facts
             scopes.add(new ObjectScope<>(snmpInterface)
-                    .map(INTERFACE, "if-alias", (i) -> Optional.ofNullable(snmpInterface.getIfAlias()))
-                    .map(INTERFACE, "if-description", (i) -> Optional.ofNullable(snmpInterface.getIfDescr()))
-                    .map(INTERFACE, "phy-addr", (i) -> Optional.ofNullable(snmpInterface.getPhysAddr())));
+                    .map(ScopeId.INTERFACE, "if-alias", (i) -> Optional.ofNullable(i.getIfAlias()))
```

Comment:
Jut a nit: you import all the other context names statically. But not this one.

Reply:
true :-) fixed